uber/config.py

```diff
@@ -180,12 +167,39 @@ def __getattr__(self, name):
                 return True  # Defaults to unlimited stock for any stock not configured
             else:
                 return count_check < stock_setting
+        elif hasattr(_secret, name):
+            return getattr(_secret, name)
         elif name.lower() in _config['secret']:
             return _config['secret'][name.lower()]
         else:
             raise AttributeError('no such attribute {}'.format(name))
 
+
```

Comment:
In case it's not clear from the above code, we can add/override properties on this class just like we do on the main ``Config`` class.  For example, I put the following code into a plugin as a test:

```python
@Config.mixin
class ExtraConfig:
    @property
    def FOO(self):
        return 'test public property'

@SecretConfig.mixin
class ExtraSecretConfig:
    @property
    def BAR(self):
        return 'test private property'
```

and then I was able to access both properties on the global ``c`` object:

```python
>>> c.FOO
'test public property'
>>> c.BAR
'test private property'
```